doc/index.md

```diff
@@ -1,8 +1,8 @@
 # JSTP / JavaScript Transfer Protocol
```

Comment:
`doc/index.md` doesn't really matter, it is completely obsolete and will be rewritten away from scratch when we get to docs, so I didn't touch it intentionally, but since you've already done it, that's okay, it won't hurt to update it too.